core/actionProxy/Dockerfile

```diff
@@ -19,5 +19,5 @@ ADD actionproxy.py /actionProxy/
 RUN mkdir -p /action
 ADD stub.sh /action/exec
 RUN chmod +x /action/exec
-
+EXPOSE 8080
```

Comment:
since we didn't address the proxy issue yet for the local container tests, is exposing the port in the runtime containers necessary?

Reply:
not necessary